1config-ui/resources/public/css/style.css

```diff
@@ -85,13 +127,16 @@ body {
 select.ui.dropdown {
     font-family: 'Cardo', serif;
     font-size: 14px;
-    /*color: lightgrey;*/
 }
 
 .ui.divider {
     text-transform: none;
 }
 
+.ui.segment {
+    margin-left: 19px;
+}
+
```

Comment:
this change is required for this indentation 
<img width="636" alt="Screenshot 2019-12-27 at 08 07 14" src="https://user-images.githubusercontent.com/3762201/71508528-0bf8c880-2880-11ea-8dcd-c01a1d17abed.png">